### ISSUE TITLE
lldb cant unwind CallSignalHandlerWrapper on arm

### ISSUE THREAD

**Issue by chunseoklee** [dotnet/coreclr#13099]
lldb on rpi2(ubuntu1604) cant unwind `CallSignalHandlerWrapper`.

```
tizen@tizen-desktop ~/c/file> ~/js2/jenkins-slave/workspace/llvm/build-rpi2/bin/lldb corerun  -c  /tmp/core_corerun.9069
(lldb) target create "corerun" --core "/tmp/core_corerun.9069"
Core file '/tmp/core_corerun.9069' (arm) was loaded.
(lldb) bt
* thread #1: tid = 0, 0x767e7ecc libcoreclr.so`VirtualCallStubManager::predictStubKind(this=<unavailable>, stubStartAddress=<unavailable>) + 104 at virtualcallstubcpu.hpp:348, name = 'corerun', stop reason = signal SIGSEGV
  * frame #0: 0x767e7ecc libcoreclr.so`VirtualCallStubManager::predictStubKind(this=<unavailable>, stubStartAddress=<unavailable>) + 104 at virtualcallstubcpu.hpp:348
    frame #1: 0x767ed1d6 libcoreclr.so`VirtualCallStubManager::getStubKind(this=<unavailable>, stubStartAddress=<unavailable>) + 22 at virtualcallstub.h:332
    frame #2: 0x767ea1b4 libcoreclr.so`VirtualCallStubManager::FindStubManager(stubAddress={PCODE}, wbStubKind={VirtualCallStubManager::StubKind}) + 56 at virtualcallstub.cpp:998
    frame #3: 0x768847e2 libcoreclr.so`IsSafeToHandleHardwareException(_CONTEXT*, _EXCEPTION_RECORD*) [inlined] IsIPinVirtualStub(f_IP={PCODE}) + 20 at exceptionhandling.cpp:5149
    frame #4: 0x768847ce libcoreclr.so`IsSafeToHandleHardwareException(contextRecord=<unavailable>, exceptionRecord=<unavailable>) + 102 at exceptionhandling.cpp:5174
    frame #5: 0x76a3e150 libcoreclr.so`SEHProcessException(exception={PAL_SEHException}) + 52 at seh.cpp:265
    frame #6: 0x76a3f14c libcoreclr.so`common_signal_handler(code=<unavailable>, siginfo=<unavailable>, sigcontext=<unavailable>, numParams=<unavailable>) + 200 at signal.cpp:839
    frame #7: 0x76a3f04e libcoreclr.so`::signal_handler_worker(code=<unavailable>, siginfo={siginfo_t}, context=<unavailable>, returnPoint={SignalHandlerWorkerReturnPoint}) + 78 at signal.cpp:426
    frame #8: 0x76a6964a libcoreclr.so`CallSignalHandlerWrapper0 + 10
(lldb)
```
Regarding https://github.com/dotnet/coreclr/pull/9650, arm/signalhandlerwrapper.S should be patched like:
```
modified   src/pal/src/arch/arm/callsignalhandlerwrapper.S
@@ -18,11 +18,12 @@ C_FUNC(SignalHandlerWorkerReturnOffset\Alignment):
 // address set to SignalHandlerWorkerReturn during SIGSEGV handling.
 // It enables the unwinder to unwind stack from the handling code to the actual failure site.
 NESTED_ENTRY CallSignalHandlerWrapper\Alignment, _TEXT, NoHandler
-    sub     sp, sp, #(8 + \Alignment) // red zone + alignment
-    stmfd   sp!, {r7, lr}
-    bl      EXTERNAL_C_FUNC(signal_handler_worker)
+    alloc_stack     (8 + \Alignment) // red zone + alignment
+    PROLOG_PUSH  "{r7, lr}"
+        bl      EXTERNAL_C_FUNC(signal_handler_worker)
 LOCAL_LABEL(SignalHandlerWorkerReturn\Alignment):
-    ldmfd   sp!, {r7, lr}
+    EPILOG_POP "{r7, lr}"
+    free_stack (8 + \Alignment)    
     bx      lr
 NESTED_END CallSignalHandlerWrapper\Alignment, _TEXT
```

**Comment by chunseoklee** [comment]
@janvorli PTAL

**Comment by janvorli** [comment]
@chunseoklee thank you! Can you please create a PR for that? The change looks good.

**Comment by RussKeldorph** [comment]
I assume this should have been closed when #13114 was merged.  Re-open if I misunderstood.
